api/models/Post.js

```diff
@@ -150,11 +156,32 @@ module.exports = bookshelf.Model.extend({
   },
 
   pushCommentToSockets: function (comment) {
-    this.sendToSubscribedSockets('commentAdded', comment)
+    var postId = this.id
+    this.sendToSubscribedSockets(`posts/${postId}`, 'commentAdded', comment)
+  },
+
+  pushMessageToSockets: function (message, userIds) {
+    var postId = this.id
+    const excludingSender = userIds.filter(id => id !== message.user_id.toString())
```

Comment:
toString() shouldn't be necessary here unless some funny business is happening upstream